mLRS/Common/esp-lib/esp-spi.h

```diff
@@ -14,39 +14,26 @@
 
 //-- select functions
 
-#ifndef SPI_SELECT_PRE_DELAY
-  #define SPI_SELECT_PRE_DELAY
-#endif
-#ifndef SPI_SELECT_POST_DELAY
-  #define SPI_SELECT_POST_DELAY
-#endif
-#ifndef SPI_DESELECT_PRE_DELAY
-  #define SPI_DESELECT_PRE_DELAY
-#endif
-#ifndef SPI_DESELECT_POST_DELAY
-  #define SPI_DESELECT_POST_DELAY
-#endif
-
-
-#ifdef SPI_CS_IO
-
-IRAM_ATTR static inline void spi_select(void)
+static inline IRAM_ATTR void spi_select(void)
 {
-    SPI_SELECT_PRE_DELAY;
-    GPOC = (1 << SPI_CS_IO); // digitalWrite(SPI_CS_IO, LOW); // CS = low
-    SPI_SELECT_POST_DELAY;
+#if defined(ESP32)
+  SPI.beginTransaction(SPISettings(SPI_FREQUENCY, MSBFIRST, SPI_MODE0)); 
```

Comment:
instead of the beginTransaction one could try to just use the wait thing from it ... we should not need to set the other stuff each time anew ... might make it alos faster ... 
the settings should/could go then to init

Reply:
I think we should only need this: https://github.com/espressif/arduino-esp32/blob/master/cores/esp32/esp32-hal-spi.c#L1118-L1122
and the SPI_MUTEX of course
but since we have an ESP32, we should not even need this, but only the SPI_MUTEX ...
might be worthwhile to try

Reply:
ah, this might be what we want: void spiSimpleTransaction(spi_t * spi)
https://github.com/espressif/arduino-esp32/blob/master/cores/esp32/esp32-hal-spi.c#L1125

Reply:
elrs somehow doesn't need any of this ...

Reply:
Updated in latest to not use beginTransaction